### DISCUSSION TITLE
misc questions/requests

### DISCUSSION THREAD

**Discussion by JeanQ2020** [GMOD/jbrowse-components#4322] [General]
With Colin's kind help, we had our jbrowse up running. Our users are very happy with its functionalities. Now we come for more...

1. can jbrowse support BCF format for variants? Many of users generate this type of format (BCF is just different compressed format but also has index and supported by bcftools)
2. we noticed that SV inspector took a very long time loading SV data and timeout often. Is there anyway to improve the loading speed? 

Thank you for all your help!

Jean

**Comment by cmdcolin** [comment]
1. we do not have support for the BCF format. it would probably be a somewhat large undertaking to add support for it i believe. it's a good idea though
2. the sv inspector slows down a lot when it loads large-ish data files, but we have a PR in progress that helps to improve the speed/scalability of the sv inspector. if you are interested in checking out the beta branch let me know...it is here but it probably needs one or two more tweaks before you could test it https://github.com/GMOD/jbrowse-components/pull/4047

**Comment by JeanQ2020** [comment]
@cmdcolin thank you so much Colin for your kind suggestions. Yes, I'm interested in checking out the beta branch. Do you want to schedule an office hour like last time to test and do some tweaks?

**Comment by cmdcolin** [comment]
sure thing, there is a new office hours link here, feel free to schedule one https://calendar.app.google/1AYZkNCQNmwdY2R26
